src/conversion/buildviewconverter.js

```diff
@@ -141,11 +141,21 @@ class ViewConverterBuilder {
 	 * @param {String|RegExp} [value] View attribute value.
 	 * @returns {module:engine/conversion/buildviewconverter~ViewConverterBuilder}
 	 */
-	fromAttribute( key, value = /.*/ ) {
+	fromAttribute( key, value = /.+/ ) {
```

Comment:
Hmmm, I haven't thought of this. This is a good point. I agree that this change was wrong.